test/test.js

```diff
@@ -137,6 +137,17 @@ describe('ScssCommentParser', function () {
       });
     });
 
+    describe('css rule', function(){
+      it('should detect it', function(){
+        var context = parser.contextParser(getContent('rule.test.scss'));
+        assert.deepEqual(context, {
+          type: 'css',
+          name: '.foo',
+          value: 'font-weight: bold;\n  color: red;\n  .bar {\n    color: blue;\n  }'
```

Comment:
(It looks like the trailing `}` is there, but it's not -- that's actually part of a nested rule.)